packages/core/state/selection/reducer.ts

```diff
@@ -101,6 +103,10 @@ export default makeReducer<SelectionStateBranch>(
         [SET_FILE_FILTERS]: (state, action) => ({
```

Comment:
Setting the action type here will allow you to avoid the `: any` declaration on line 107 while mapping
```suggestion
        [SET_FILE_FILTERS]: (state, action: SetFileFiltersAction) => ({
```

Reply:
looks like this wasn't done